src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/quick-start-nrql-condition.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Write your first NRQL alerts condition
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+  - Alert conditions
+  - NRQL
+metaDescription: How to start writing NRQL alerts conditions with a step-by-step user case.
+---
+
+Use [NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to create alerts conditions with ease. That way, you can define what you need to be notified about.
+
+Your condition only needs two attributes: a query and a threshold. Your query determines what you want to be notified about, and the threshold determines when you’ll be notified. In our example, you want to be notified about a host’s CPU usage (query) when usage is above 70% (threshold).
+
+Follow these steps to write your first alerts condition using the NRQL query builder. Once you're done, you'll have a working alerts condition.
+
+## Prerequisites [#pre-nrql]
+
+If you haven't already, [create a New Relic account](https://newrelic.com/signup) before you start. It's free, forever. Then, send your data into New Relic to query. Policies also hold alerts conditions. You can either create a [policy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) before starting a query or while filling out the **Create Alert** form used later.
+
+## Step 1: Write your query [#step1-nrql]
+
+Determine what alerts condition you want to query before continuing. You can create an alerts condition from any NRQL query. Find the NRQL query builder either at the top or bottom of the New Relic One platform.
+
+![A screenshot showing where you can find the NRQL query builder in New Relic One.](./images/find-builder.png "A screenshot showing where you can find the NRQL query builder in New Relic One.")
+<figcaption>
+  The graph icon at the top includes the **Create alerts** button that will be used in this example. Click the **Open in Query builder** button after using the bottom NRQL query choice to reach the same page.
```

Comment:
I'd delete this first sentence. 
I'd rewrite the second sentence to simplify it a bit: "Click the **Query your data** button open the query builder to start writing a NRQL query."
And then add something after the second sentence like: "Once you've written a valid query, you can create an alert from it."